punchcut/src/source/glyf/math.rs

```diff
@@ -0,0 +1,135 @@
+//! Various fixed point math functions used in TrueType scaling and hinting.
```

Comment:
If I am not mistaken, a couple of functions in this file are not used yet and could be landed separately, haven't found references for `muldiv_no_round` or `hypot` for example.

Reply:
`hypot` is used for scaled component offsets but `muldiv_no_round` and a few others are only used in the hinting code. I can remove these and restore them in a subsequent PR.